### PR TITLE
Add debug messages for HILS

### PR DESCRIPTION
**Pull request by seki-hiro**
## Overview
WIP: Update HILS environment

## Issue
NA

## Details
- Add debug code
- Modify comment

##  Validation results
NA

## Scope of influence
NA

## Supplement
NA

## Note
NA

### CODE REVIEW

File: src/Interface/HilsInOut/Ports/HilsI2cTargetPort.cpp [hunk 1]
```diff
@@ -59,6 +59,12 @@ int HilsI2cTargetPort::Receive()  // from I2C-USB Target converter
   int received_bytes = ReadRx(rx_buf, 0, kDefaultCmdSize);
   if (received_bytes > kDefaultCmdSize) return -1;
 
+  for (int i = 0; i < received_bytes; i++)
```
Comment (200km) [inline src/Interface/HilsInOut/Ports/HilsI2cTargetPort.cpp:62]: Please add `ifdef HILS_PORT_MANAGER_SHOW_DEBUG_DATA` same as others.

**Review by 200km — CHANGES_REQUESTED** [review]
If you already finish the update, please remove `WIP`.

**Comment by 200km** [comment]
Please change the title of this PR like `Add debug messages for HILS`.

File: src/Interface/HilsInOut/Ports/HilsI2cTargetPort.cpp (hunk 1 above)
Comment (seki-hiro) [reply src/Interface/HilsInOut/Ports/HilsI2cTargetPort.cpp:64]: WIPを外し、修正を終えました。

**Review by 200km — APPROVED** [review]

**Pull request merged**